core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -121,7 +121,7 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     // components may change without re-rendering each of the components,
     // in such cases, it is sufficient to update the ckeditor instance manually
     // with the latest value.
-    let value = this.value;
+    let value = this.value ? this.value : '';
```

Comment:
The current explanation doesn't really explain why the value passed in could be null/undefined in the first place. It would be good to string things up so that that case never happens, so if it's a temporary thing I think we should (a) explain what the problem is, (b) file an issue for the long-term fix, and (c) link to the issue from here.

Reply:
Added a more descriptive comment.